scripts/build/libc/glibc.sh

```diff
@@ -170,6 +170,9 @@ do_libc_backend() {
                              libc_target="${target}"
 
         CT_Popd
+        if [ "${multi_dir}" != "." ]; then
+            CT_EndStep
```

Comment:
This missing EndStep is actually caused by d28a3dab (glibc.sh: Use --print-multi-os-directory)
which does remove a EndStep and thus causes the imbalance.